src/cmd/api/controllers/video_archive_handler.go

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Sogilis/Voogle/src/pkg/uuidgenerator"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/db/dao"
+	"github.com/Sogilis/Voogle/src/cmd/api/models"
+)
+
+type VideoArchiveVideoHandler struct {
+	VideosDAO *dao.VideosDAO
+	UUIDGen   uuidgenerator.IUUIDGenerator
+}
+
+// VideoArchiveVideoHandler godoc
+// @Summary Archive video
+// @Description Archive video
+// @Tags status
+// @Accept plain
+// @Produce plain
+// @Param id path string true "Video ID"
+// @Success 200 {string} string "OK"
+// @Failure 400 {object} object
+// @Failure 404 {object} object
+// @Failure 500 {object} object
+// @Router /api/v1/videos/{id}/archive [put]
+func (v VideoArchiveVideoHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
We just modify the documentation for others handler, can you modify yours according video_info_handler.go for example? And same thing for unarchive.